dspot/src/main/java/eu/stamp_project/utils/report/output/selector/coverage/json/TestClassJSON.java

```diff
@@ -43,9 +43,9 @@ public boolean addTestCase(TestCaseJSON testCaseJSON) {
 
 	public String toString() {
 		JSONObject subJson = new JSONObject()
-                  .put("initialCoverage", this.initialInstructionCovered)
```

Comment:
why that change? it seems unrelated.

Reply:
Hi :) , I wanted to have it already capitalized at first letter so I don't need to capitalize it later. Also since Pitmutant use int so this make the code shorter to also have all the values as int, so I only need to use .getInt() later in MongodbManager.

Reply:
Is that required for this PR?

Reply:
Hi :) , I have revert the changes to that part in the latest commit.